administrator/components/com_media/controllers/file.json.php

```diff
@@ -56,6 +56,7 @@ public function upload()
 		// Get some data from the request
 		$file   = $this->input->files->get('files', '', 'array');
 		$folder = $this->input->get('folder', '', 'path');
+		$canOverwrite = $this->input->getBool('allowovewrite', false);
```

Comment:
Also this file needs some major refactoring, as I found some inconsistent behaviour in the method that checks the filename